Sources/Sentry/SentryClient.m

```diff
@@ -634,6 +643,29 @@ - (void)recordLost:(BOOL)eventIsNotATransaction reason:(SentryDiscardReason)reas
     }
 }
 
+- (void)attachScreenshots:(SentryScope *)scope
+{
+#if SENTRY_HAS_UIKIT
+    if (!scope || !self.options.attachScreenshot)
+        return;
+
+    NSArray *shots = [SentryDependencyContainer.sharedInstance.screenshot appScreenshots];
+
+    for (int i = 0; i < shots.count; i++) {
```

Comment:
Can `shots` be null here?

Reply:
Yes, but this is not a problem for obj-c